user/views.py

```diff
@@ -11,14 +10,8 @@ class CreateUserView(generics.CreateAPIView):
     serializer_class = UserSerializer
```

Comment:
you don't use api_settings and ObtainAuthToken and AuthTokenSerializer, so you can remove these imports